gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/GravatarQuickEditorPage.kt

```diff
@@ -33,14 +43,20 @@ internal fun GravatarQuickEditorPage(
     onDismiss: (dismissReason: GravatarQuickEditorDismissReason) -> Unit = {},
 ) {
     val navController = rememberNavController()
+    val editorViewModel: AvatarPickerViewModel = viewModel(
+        factory = AvatarPickerViewModelFactory(
+            gravatarQuickEditorParams = gravatarQuickEditorParams,
+            handleExpiredSession = true,
+        ),
+    )
```

Comment:
⚠️ There's a slight change in behavior here. This will make the `AvatarPickerViewModel` service QE relaunches. You can see that when you close/open the QE, there's no initial loading. 

I wonder if this would be problematic for apps that provide the token. In a scenario where the user changes but the ViewModel survives and persists the previous data.

Reply:
My suggestion would be to opt for a separate ViewModel anyway. So maybe it's not worth thinking about how to fix that.